addons/pos_self_order/models/pos_payment_method.py

```diff
@@ -0,0 +1,8 @@
+from odoo import models
+
+
+class PosPaymentMethod(models.Model):
+    _inherit = "pos.payment.method"
+
+    def payment_request_from_kiosk(self, order):
+        pass
```

Comment:
I find this strange, at least we should put a comment saying that it is meant to be overriden by payment_method provider modules